ports/raspberrypi/boards/pimoroni_picosystem/mpconfigboard.mk

```diff
@@ -0,0 +1,11 @@
+USB_VID = 0x16D0
```

Comment:
~~http://www.linux-usb.org/usb.ids suggests that USB VID 16D0 is "MCS", which doesn't seem appropriate here?~~
Ahhh, I just found https://www.mcselec.com/index.php?page=shop.product_details&flypage=shop.flypage&product_id=92&option=com_phpshop&Itemid=1 which makes sense :smile:

Reply:
Right, Pimoroni doesn't have their own USB VID.